frontend/services/api/meta.ts

```diff
@@ -66,3 +66,25 @@ export const removeStarredMeta = (metaRecordKey) => {
 export const overrideStarredMeta = (starredMeta) => {
 	return api.put('starredMeta', starredMeta);
 };
+
+/**
+ * Get mesh IDs by query
+ * @param teamspace
+ * @param modelId
+ * @param rules
+ * @param revision
+ */
+export const getMeshIDsByQuery = (teamspace, modelId, rules, revision = 'master/head') => {
+	return api.post(`${teamspace}/${modelId}/revision/${revision}/meta/rules?meshids=true`, rules);
+};
+
+/**
+ * Get Metadata by query
+ * @param teamspace
+ * @param modelId
+ * @param rules
+ * @param revision
+ */
+export const getMetadataByQuery = (teamspace, modelId, rules, revision = 'master/head') => {
```

Comment:
Is this used?

Reply:
Yea I'm not sure if we'll ever use it on the front end. This was more added for API users in mind